python/lsst/analysis/tools/actions/plot/propertyMapPlot.py

```diff
@@ -41,6 +45,110 @@
 _LOG = logging.getLogger(__name__)
 
 
+def getZoomedExtent(fullExtent, n):
+    """Get zoomed extent centered on the original full plot.
+
+    Parameters
+    ----------
+    fullExtent : `tuple` [`float`]
+        The full extent defined by (lon_min, lon_max, lat_min, lat_max):
+
+        * ``lon_min``
+            Minimum longitude of the original extent (`float`).
+        * ``"lon_max"``
+            Maximum longitude of the original extent (`float`).
+        * ``lat_min``
+            Minimum latitude of the original extent (`float`).
+        * ``"lat_max"``
+            Maximum latitude of the original extent (`float`).
+
+    n : `float`, optional
+        Zoom factor; for instance, n=2 means zooming in 2 times at the
+        center. If None, the function returns None.
+
+    Returns
+    -------
+    `tuple` [`float`]
+        New extent as (new_lon_min, new_lon_max, new_lat_min, new_lat_max).
+    """
+    if n is None:
+        return None
```

Comment:
Why is this necessary?

Reply:
We use this function within a loop where we occasionally pass `None` to indicate that the full original plot should be displayed. Skyproj interprets `extent=None` as a signal that no zooming is required and renders the complete plot.